### ISSUE TITLE
AZGTi mount control always moves on the same direction

### ISSUE THREAD

**Issue by miceno** [indilib/indi#1502]
**Describe the bug**
Using Kstars telescope control to move an AZGTi with driver `indi_azgti_telescope`, whatever the arrow button I click, it always moves in the same direction. If I click left, it moves down, if I click right, it continues moving down.

**To Reproduce**
Exact steps to reproduce the behavior.
1.  Open Ekos, and open the profile that includes the AZGTi driver `indi_azgti_telescope`
2. Connect via Wifi to the mount. I am using AZGTi station mode, and the mount connects to an [astroberry](https://www.astroberry.io/) hotspot.
2. Open the Mount Control Panel.
3. Move the mount by clicking using whatever direction arrow.
4. Same behaviour if you move the mount from the mount panel on INDI control panel.

AZGTi in equatorial mode with latest firmware: "AZGTi Mount, Right Arm, AZ/EQ Dual Mode, Version 3.26"

If at the same time I connect to the mount using SynScanPro (by Wifi, not using the Ekos software), the mount moves properly.

The problem is only reproduced while manually moving the mount. GOTO works.

**Expected behavior**
The mount should move according to the arrow clicked on the Mount Control Panel or the direction clicked on the INDI control panel.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: astroberry (latest 2.0.3).
 - Kstars: 3.5.3, but also happened in 3.5.2

**Log Files**
Reproduced twice while observing during the night, pending to reproduce during daytime to collect logs.

**Comment by knro** [comment]
I tested yesterday and I could not reproduce this behavior.

**Comment by miceno** [comment]
Sorry for not providing more details, but it is hard... I am using it without any problem, and then suddenly, the arrows on the mount control only work on one direction, it is usually just one direction that is not working, not both directions at the same time.

It happened again tonight, so I had to use the SynScan Pro app instead of Ekos/Kstars to move the telescope.

One of those times, I managed to solved it, but I am not sure what I did 😞 , I think it was related to syncing or activating something on the tracking, but I don't know exactly what I did. 

I will try to get logs in the next session, but it was a little bit frustrating, I only have 1-2 hours in the evening to play with it, I will try to test it during the day.
